src/tests/system/tests/test_passkey.py

```diff
@@ -3,11 +3,67 @@
 
 :requirement: passkey
 
+Passkeys allow users to authenticate without having to enter a username or password,
+or provide any additional authentication factor.
+This technology aims to replace legacy authentication mechanisms such as passwords.
+
+The objective is to use a passkey to locally authenticate a user against centralized identity management system.
+For that purpose an integration with a server like AD/Samba or IPA or LDAP is needed.
+
 The passkey solution only enables to authenticate in a system where the
-FIDO2 key is connected physically.This could be su login, the GUI,
+FIDO2 key is connected physically.This could be su login, the GDM login,
```

Comment:
I think we should just drop SSH for now, I feel that this will change eventually. Just "The passkey is another way to authenticate as the user, using a physical token".